1-hello-world/package.json

```diff
@@ -6,7 +6,8 @@
   "private": true,
   "scripts": {
     "start": "node app.js",
-    "test": "ava -t 30s test/*.test.js | tap-dot",
+    "e2e": "rm -rf node_modules/ &&  samples test build --app --deploy --timeout 20m --builderProject $GCLOUD_PROJECT",
```

Comment:
`--app` might be redundant because you've defined a `test` script that actually runs `samples test app`. Passing the `--app` flag to `build` would make `samples test app` get run twice.